patches/chrome-installer-linux-BUILD.gn.patch

```diff
@@ -36,3 +36,10 @@ index fbabd6c3414f5cb69cf4c7ab6ef5f957cb391376..ab6bf7f0d65ae03730d977b17782d52a
      ]
    }
  
+@@ -524,3 +523,6 @@ linux_package("beta") {
+ linux_package("unstable") {
+   channel = "unstable"
+ }
++linux_package("nightly") {
```

Comment:
can this be added in a brave build file or does it have to be here?

Reply:
To add this in a brave build, we need to copy `linux_package` template in here to us.
So, I added this here to avoid copy.